### PR TITLE
Add CloudApp.app v3.7.0

### PR DESCRIPTION
**Pull request by pstauffer**
Adding a new cask for CloudApp in version 3.7.0

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Read the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md). It must be followed, to avoid duplicates.

[This already exists](https://github.com/caskroom/homebrew-cask/blob/master/Casks/cloud.rb).

File: Casks/cloudapp.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'cloudapp' do
+  version '3.7.0'
+  sha256 '29b6ad9938c76e9007a66f156922777fd5ec3deb6ac6d3b3a6d147142d07cb9a'
+
+  url "https://s3.amazonaws.com/downloads.getcloudapp.com/mac/CloudApp-#{version}.dmg"
+  name 'CloudApp'
+  homepage 'http://www.getcloudapp.com/'
+  license :commercial
```
Comment (vitorgalvao) [inline Casks/cloudapp.rb]: It’s not commercial, as you do not pay for the app.

File: Casks/cloudapp.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'cloudapp' do
```
Comment (vitorgalvao) [inline Casks/cloudapp.rb]: This header is wrong. We no longer use `:v1 =>` (it is removed).

**Comment by pstauffer** [comment]
Hi there,

thanks for the feedback and the link to the already existing cask. In my oppinion is the already existing cask token wrong, because the app is named "CloudApp", so the cask token should be "cloudapp" and not "cloud".
But anyway, I will use your feedback, if I create another cask.

Thanks and Greets
Pascal

**Comment by vitorgalvao** [comment]
> In my oppinion is the already existing cask token wrong, because the app is named "CloudApp", so the cask token should be "cloudapp" and not "cloud".

This exact case has been discussed before. We need consistency and strict rules, because we know what a repo filled with duplication looks like, and it is not good. “App” is a common suffix that doesn’t matter most of the time and makes more sense to be removed. CloudApp isn’t special (and has an arguably weak name) and is the only app where this rule causes some occasional doubts. Having duplicates isn’t worth the exception.

**Pull request closed without merging**